cocotb/regression.py

```diff
@@ -493,20 +506,6 @@ class TestFactory(object):
     """
 
     def __init__(self, test_function, *args, **kwargs):
-        """
-        Args:
-            test_function (function): the function that executes a test.
-                Must take *dut* as the first argument.
-
-            *args: Remaining arguments are passed directly to the test function.
-                   Note that these arguments are not varied. An argument that
-                   varies with each test must be a keyword argument to the
-                   test function.
-            *kwargs: Remaining keyword arguments are passed directly to the test function.
-                   Note that these arguments are not varied. An argument that
-                   varies with each test must be a keyword argument to the
-                   test function.
-        """
```

Comment:
Does this render incorrectly if left here?

Reply:
No, but it relied on the non-intuitive initial line break immediately after the triple opening quote .